### PR TITLE
a start at russian thanks to @krava2008

### PR DESCRIPTION
**Pull request by kevinkreiser**

### CODE REVIEW

File: locales/ru-RU.json [hunk 1]
```diff
@@ -0,0 +1,1003 @@
...
+        "1": ["Вы прибываете к 32 o6 Powelton Avenue."],
+        "2": ["Место вашего назначения будет слева.", "Место вашего назначения будет справа."],
+        "3": ["Lancaster Brewing Company будет справа."]
+      }
+    },
+    "destination_verbal": {
+      "phrases": {
+        "0": ["Вы прибыли по месту назначения."],
+        "1": ["Вы прибыли к <DESTINATION>."],
+        "2": ["Местоназначение слева.", "Местоназначение слева."],
```
Comment (kevinkreiser) [inline locales/ru-RU.json]: @krava2008 can you fix this phrase. there should be only one in here and it should have `<RELATIVE_DIRECTION>`

**Comment by dnesbitt61** [comment]
:shipit:

File: locales/ru-RU.json [hunk 2]
```diff
@@ -0,0 +1,1003 @@
...
+        "3": ["Lancaster Brewing Company слева."]
```
Comment (kevinkreiser) [inline locales/ru-RU.json]: @krava2008 this phrase should contain `<DESTINATION>`

File: locales/ru-RU.json (hunk 1 above)
Comment (kevinkreiser) [inline locales/ru-RU.json]: is this correct? `"2": ["Местоназначение будет <RELATIVE_DIRECTION>."],`

File: locales/ru-RU.json (hunk 2 above)
Comment (kevinkreiser) [inline locales/ru-RU.json]: is this correct? "3": ["<DESTINATION> будет <RELATIVE_DIRECTION>."]

File: locales/ru-RU.json [hunk 3]
```diff
@@ -100,10 +100,10 @@
...
     "destination_verbal": {
       "phrases": {
-        "0": ["Вы прибыли по месту назначения."],
-        "1": ["Вы прибыли к <DESTINATION>."],
-        "2": ["Местоназначение слева.", "Местоназначение слева."],
-        "3": ["Lancaster Brewing Company слева."]
+        "0": "Вы прибыли по месту назначения.",
+        "1": "Вы прибыли к <DESTINATION>.",
+        "2": "Местоназначение будет <RELATIVE_DIRECTION>.",
+        "3": "<DESTINATION> будет <RELATIVE_DIRECTION>."
```
Comment (kevinkreiser) [inline locales/ru-RU.json]: @krava2008 are these correct?

**Comment by kevinkreiser** [comment]
@krava2008 what is the correct format for dates in russian? the internet seems to suggest, as i would expect: `dd.mm.yyyy` but it seems posix `ru_RU` is giving `dd/mm/yyyy`

**Comment by kevinkreiser** [comment]
this should be mergable as soon as @krava2008 signs off on the `destination_verbal` changes i made above

**Pull request merged**